internal/util/token.go

```diff
@@ -0,0 +1,124 @@
+package util
+
+import (
+	"net/http"
+	"net/http/cookiejar"
+	"net/url"
+
+	log "github.com/sirupsen/logrus"
+
+	"fmt"
+	"github.com/pkg/errors"
+	"golang.org/x/net/html"
+)
+
+type environment struct {
+	authURL     string
+	redirectURL string
+}
+
+const (
+	stage      = "stage"
+	prod       = "prod"
+	json_token = "token_json"
+)
+
+var (
+	environments = make(map[string]environment, 2)
+)
+
+func init() {
+	environments[stage] = environment{
+		authURL:     "https://auth.prod-preview.openshift.io/api/login",
+		redirectURL: "https://jenkins.prod-preview.openshift.io",
+	}
+
+	environments[prod] = environment{
```

Comment:
I don't like harcoding these URLs in the git repo

Reply:
What's the issue? It is not really any secret URL, is it? And more importantly, what's the alternative? To pass the URLs as command line arguments. To a certain degree, it will diminish the value/convenience of this tool.